File: src/test/java/day03/GearRatiosTest.java
package day03;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class GearRatiosTest {

    @Test
    void firstStar() {
        GearRatios gearRatios = new GearRatios();
        assertEquals(4361, gearRatios.firstStar());
    }

    @Test
    void secondStar() {
        GearRatios gearRatios = new GearRatios();
        assertEquals(467835, gearRatios.secondStar());
    }
}

File: src/test/kotlin/day03/GearRatiosTest.kt
package day03

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

internal class GearRatiosTest {
    @Test
    fun firstStar() {
        assertEquals(4361, GearRatios().firstStar())
    }

    @Test
    fun secondStar() {
        assertEquals(467835, GearRatios().secondStar())
    }
}